lib/document/document-converter.js

```diff
@@ -510,7 +510,7 @@ ${faDom.css()}
       const tocPlacement = node.getAttribute('toc-placement', 'auto')
       // Add a toc in the header if the toc placement is auto (default), left or right
       const hasTocPlacementHeader = tocPlacement === 'auto' || tocPlacement === 'left' || tocPlacement === 'right'
-      if (node.hasSections() && node.hasAttribute('toc') && (hasTocPlacementHeader || this.hasTitlePage(node))) {
+      if (node.hasSections() && node.hasAttribute('toc') && (hasTocPlacementHeader || this.hasTitlePage(node) && tocPlacement !== 'preamble')) {
```

Comment:
I think it should be:

```suggestion
      if (node.hasSections() && node.hasAttribute('toc') && hasTocPlacementHeader) {
```

Reply:
You are right :blush: